examples/asa-vault/VaultOwner.js

```diff
@@ -9,9 +9,9 @@ dotenv.config();
   VAULTS.TestNet.algo.vaultId;
   console.log(VAULT_ID);
   const STABLECOIN = process.env.STABLE_COIN;
+  const COLLATERALTOKEN = process.env.COLLATERAL_TOKEN;
```

Comment:
```suggestion
  const COLLATERAL_TOKEN = process.env.COLLATERAL_TOKEN;
```
Just a stylistic preference